redshift_sqlalchemy/dialect.py

```diff
@@ -276,12 +279,13 @@ def visit_copy_command(element, compiler, **kw):
     return """
            COPY %(schema_name)s.%(table_name)s FROM '%(data_location)s'
            CREDENTIALS 'aws_access_key_id=%(access_key)s;aws_secret_access_key=%(secret_key)s%(session_token)s'
-           CSV
```

Comment:
cool i'm going to submit a new pr that has current master merged into, let's discuss there